content/posts/2025-03-10-deepseek-r1.md

```diff
@@ -0,0 +1,221 @@
+- **Object Storage with AI Runtime**: Users have the option to store model weights in object storage services like Amazon S3 or Google Cloud Storage (GCS). In this case, the AIbrix AI runtime will automatically download the model to the host volume. This approach offers flexibility and scalability, leveraging the advantages of object storage for storing large amounts of data.
+- **Local Disk**: For local disk storage, an additional process is required to download the model weights to the local disk. We assume that a local volume is available and can be successfully mounted to the pod. This option may be suitable for environments where local storage offers performance benefits or when there are specific security or latency requirements.
+
+| Storage Options                                 | Description                              | Sample Files                           |
+| ----------------------------------------------- |------------------------------------------|----------------------------------------|
+| HuggingFace                                     | no volume needed                         | [Link](https://github.com/vllm-project/aibrix/tree/main/samples/deepseek-r1/deepseek-r1-huggingface.yaml) |
+| Persistent Volume                               | models volume, PVC                       | [Link](https://github.com/vllm-project/aibrix/tree/main/samples/deepseek-r1//deepseek-r1-pvc.yaml)         |
+| Object Storage(S3 / GCS) with AIBrix AI Runtime | models volume, HostPath                  | [Link](https://github.com/vllm-project/aibrix/tree/main/samples/deepseek-r1//deepseek-r1-ai-runtime.yaml)  |
+| Local Disk                                      | models volume, HostPath + InitContainer  | [Link](https://github.com/vllm-project/aibrix/tree/main/samples/deepseek-r1//deepseek-r1-local-nvme.yaml)  |
+
+### High Performance Network
+
+To leverage RDMA to achieve the best performance in network communication, we need to configure the pod configuration to use RDMA.
+`k8s.volcengine.com/pod-networks` is configured in the annotation like below and `vke.volcengine.com/rdma: "8"` is needed at the pod resource level.
+This is just one example on volcano engine cloud, You need to make corresponding change based on your own cloud environment.
+
+```yaml
+  k8s.volcengine.com/pod-networks: |
+    [
+      {
+        "cniConf":{
+            "name":"rdma"
+        }
+      },
+      ....
+      {
+        "cniConf":{
+            "name":"rdma"
+        }
+      }
+    ]
+```
+Besides that, we also need `IPC_LOCK` and Share Memory Support.
+
+```yaml
+  securityContext:
+    capabilities:
+      add:
+      - IPC_LOCK
+```
+
+## Installation
+
+You need AIBrix v0.2.1, which has been proven effective for our multi-nodes deployment needs. 
```

Comment:
Line 106
"You need AIBrix v0.2.1, which has been proven effective for our multi-nodes deployment needs"
=> "AIBrix v0.2.1 is required for multi-node deployment."